stl/inc/ranges

```diff
@@ -827,11 +827,13 @@ namespace ranges {
                 requires (_Choice<_Rng>._Strategy != _St::_None)
             _NODISCARD constexpr auto operator()(_Rng&& _Range) const noexcept(_Choice<_Rng>._No_throw) {
                 // clang-format on
-                if constexpr (_Choice<_Rng>._Strategy == _St::_View) {
+				constexpr _St _Strat = _Choice<_Rng>._Strategy;

```

Comment:
There is strange formatting here

Reply:
Thang you for noticing, it's because of tabs instead of spaces, I'd pushed fix before saw your comment about superfluous newline, I'll fix it